modules/op-disabling-automatic-creation-of-rbac-resources.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// */openshift_pipelines/installing-pipelines.adoc
+[id="op-disabling-automatic-creation-of-rbac-resources_{context}"]
+= Disabling the automatic creation of RBAC resources
+
+After default installation, the {pipelines-title} Operator creates multiple Role Based Access Control (RBAC) resources for all namespaces in the cluster. Among them, the `pipelines-scc-rolebinding` SCC role binding resource is a potential security issue, because the associated `pipelines-scc` Security Context Constraint has `RunAsAny` privilege. 
```

Comment:
```suggestion
After default installation, the {pipelines-title} Operator creates multiple Role Based Access Control (RBAC) resources for all namespaces(except openshift-* namespaces) in the cluster. Among them, the `pipelines-scc-rolebinding` SCC role binding resource is a potential security issue, because the associated `pipelines-scc` Security Context Constraint has `RunAsAny` privilege. 
```

Reply:
For ref: https://github.com/tektoncd/operator/blob/main/pkg/reconciler/common/common.go#L41